<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/free-icon-hand-cursor-5781782.png</normaloff>:/icon/free-icon-hand-cursor-5781782.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-450</x>
      <y>-40</y>
      <width>2401</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background-image: url(:/icon/Screenshot from 2024-07-04 10-28-03.png);
    background-position: bottom center; /* 바닥 중앙에서 시작 */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hightfiveimo">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>160</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/free-icon-hand-cursor-5781782.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hilabel">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>160</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Hi</string>
    </property>
   </widget>
   <widget class="QLabel" name="kiosklabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>561</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>120</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>KiOSK</string>
    </property>
   </widget>
   <widget class="QLabel" name="orderlabel">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>520</y>
      <width>161</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Go Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orderBtn">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>470</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0); /* 투명 배경 */
    border: none; /* 버튼 테두리 제거 */
    image: url(:/icon/free-icon-direction-1653447.png); /* 아이콘 설정 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>hightfiveimo</zorder>
   <zorder>hilabel</zorder>
   <zorder>kiosklabel</zorder>
   <zorder>orderBtn</zorder>
   <zorder>orderlabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
